### DISCUSSION TITLE
Add modded items to supply

### DISCUSSION THREAD

**Discussion by Git-Coat** [TEHE-Studios/ExpandedHelicopterEvents#227] [Q&A]
Is there any way I can add modded items to supply crates? Ive been thinking of adding a simple vaccine to a supply crate using the zombie virus vaccine mod

**Comment by Chuckleberry-Finn** [comment]
Another person other than us can write a sub-mod if they wish to add support. Examples of how to integrate other mods are present in the current code base.
